test/extended/single_node/topology.go

```diff
@@ -0,0 +1,149 @@
+package single_node
+
+import (
+	"context"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "github.com/openshift/api/config/v1"
+	exutil "github.com/openshift/origin/test/extended/util"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	"strings"
+)
+
+func getOpenshiftNamespaces(f *e2e.Framework) []corev1.Namespace {
+	list, err := f.ClientSet.CoreV1().Namespaces().List(context.Background(), metav1.ListOptions{})
+	Expect(err).NotTo(HaveOccurred())
+
+	var openshiftNamespaces []corev1.Namespace
+	for _, namespace := range list.Items {
+		if strings.HasPrefix(namespace.Name, "openshift-") {
+			openshiftNamespaces = append(openshiftNamespaces, namespace)
+		}
+	}
+
+	return openshiftNamespaces
+}
+
+func getNamespaceDeployments(f *e2e.Framework, namespace corev1.Namespace) []appsv1.Deployment {
+	list, err := f.ClientSet.AppsV1().Deployments(namespace.Name).List(context.Background(), metav1.ListOptions{})
+	Expect(err).NotTo(HaveOccurred())
+
```

Comment:
```suggestion
    return list.Items
```

You're not doing any filtering here?

Reply:
Right, no filtering. The loop looks ridiculous because it's leftover after some modifications I did that made it obsolete. Will fix. Thanks

Reply:
Fixed in 0debdebfc6